src/force-torque-estimator.cpp

```diff
@@ -459,6 +357,36 @@ namespace dynamicgraph
         getProfiler().stop(PROFILE_JOINTS_TORQUES_MOTOR_COMPUTATION);
         return s;
       }
+
+      /** Estimate the joints' positions, velocities and accelerations. */
+      DEFINE_SIGNAL_INNER_FUNCTION(q_dq_ddq, dynamicgraph::Vector)
```

Comment:
You have the option of using either the filtered q/dq signals, or the raw encoder data for calculating the torques (in the torques_wrenches internal signal). This is the reason the q_dq_ddq signal is replicated in both ft_estimation and the new entity. 

The other alternative would be to take the position signals out from the new entity and feed them to the ft_estimator. This would mean that ft_estimator is dependent on the kinematic_estimator